include/SFML/Window/ContextSettings.hpp

```diff
@@ -76,6 +81,8 @@ struct ContextSettings
     unsigned int majorVersion;      ///< Major number of the context version to create
     unsigned int minorVersion;      ///< Minor number of the context version to create
     Uint32       attributeFlags;    ///< The attribute flags to create the context with
+    std::string  vendorName;        ///< Name of the vendor of the OpenGL implementation
+    std::string  rendererName;      ///< Name of the OpenGL renderer implementation
```

Comment:
I agree with Laurent. Also the name "Context**Setting**" implies that it's something the user can set/configure.

One alternative would be a new type like [`sf::Joystick::Identification`](https://github.com/LaurentGomila/SFML/blob/master/include/SFML/Window/Joystick.hpp#L76-L83) and a `sf::Window::getGpuIdentification()` method or similar. Other ideas?